packages/web-components/src/components/ic-accordion/ic-accordion.css

```diff
@@ -0,0 +1,100 @@
+@import "../../global/normalize.css";
+
+:host {
+  display: block;
+}
+
+:host(.light) ic-typography,
+:host(.light) .expanded-content,
+:host(.light) .icon-container,
+:host(.light) .expand-chevron {
+  color: var(--ic-architectural-white);
+}
+
+:host(.disabled) ic-typography,
+:host(.disabled) .icon-container,
+:host(.disabled) .expand-chevron {
+  color: var(--ic-architectural-500);
+}
+
+.section-button.small {
+  padding: var(--ic-space-xxs) var(--ic-space-xs);
+}
+
+.section-button.large {
+  padding: var(--ic-space-sm) var(--ic-space-xs);
+}
+
+:host(:first-child) {
+  border-top: 1px solid var(--ic-architectural-300);
+}
+:focus {
+  outline: none;
+}
+
+.section-button {
+  background-color: transparent;
+  display: flex;
+  align-items: center;
+  width: 100%;
+  padding: var(--ic-space-xs);
+  font-weight: var(--ic-font-weight-bold);
+  border: none;
+  border-bottom: 1px solid var(--ic-architectural-300);
+}
+
+.section-button-open {
+  border-bottom: none;
+}
+
+.section-header {
+  text-align: left;
+  flex: 1 0;
+}
+
+button:hover {
+  background-color: var(--ic-action-dark-bg-hover);
+  cursor: pointer;
+}
+
+button:active {
+  background-color: var(--ic-action-dark-bg-active);
+}
+
+button:focus {
+  box-shadow: var(--ic-border-focus);
+  border-radius: var(--ic-border-radius);
+  transition: var(--ic-transition-duration-fast);
+}
+
+.icon-container {
+  margin: 0 8px 0 0;
+}
+
+.expand-chevron {
+  width: var(--ic-space-lg);
+  height: var(--ic-space-lg);
+  margin-left: 40px;
+  transform: rotate(90deg);
+  justify-self: end;
+}
+
+.content-expanded-chevron {
+  transform: rotate(-90deg);
+}
+
+.expanded-content {
+  height: 0;
+  overflow: hidden;
+  flex-direction: column;
+  transition: height 0.3s ease-out;
+}
+
+.expanded-content-inner {
+  padding: var(--ic-space-xs);
+}
+
+.expanded-content-open {
+  display: flex;
+  border-bottom: 1px solid var(--ic-architectural-300);
```

Comment:
```suggestion
  border-bottom: var(--ic-space-1px) solid var(--ic-architectural-300);
```

Reply:
Actually might want to check with @DBD324 that this is correct since the recent changes for updating borders to architectural 400. If the case is that it should be updated to 400, then you can use the --ic-border-default variable